### PR TITLE
[finishes #161039430] create login page

### PR DESCRIPTION
**Pull request by charitymarani**
**What does this PR do?**
Sets up UI for a user to login in

**Description of Task to be completed?**
Creating a user interface for a user to log in by entering the username and password

**How should this be manually tested?**
clone this repo and checkout to ft-login-page-161039430
Open login.html in a browser
**Any background context you want to provide?**
none

**What are the relevant pivotal tracker stories?**
#161039430

**Screenshots (if appropriate)**
screenshot 232

**Questions:**
What do you think about the design?

### CODE REVIEW

**Comment by EricOuma** [comment]
The design is awesome

**Comment by kwanj-k** [comment]
nice design

**Comment by Georgeygigz** [comment]
Nice interface Charity

**Comment by wandesky** [comment]
Looks good to me.

**Comment by KelynPNjeri** [comment]
The design is amazing but what if you changed the color of the button.

File: login.html [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+                <br/>
+                <p class="copyright">
+                    &copy; 2018 Store Manager
+                </p>
+            </div>
+        </div>
+    </div>
+</body>
+
+</html>
```
Comment (SelaDanti) [inline login.html]: Your UI design is amazing. Its good practice to put an extra line at the end of your file

Comment (Teatoller) [reply login.html]: Hi Charity,
The design has an awesome appeal and your narrations sound great.

**Comment by sharkdevs** [comment]
Hello charity, Looking at the name of the PR, its like You named it after the last Commit message. Change it to be compliant to the  [GitHub Naming Standards](https://github.com/andela/bestpractices/wiki/Git-naming-conventions-and-best-practices) 
Good work

**Comment by Paulstar200** [comment]
Nice!!!

**Comment by DennisMufasa** [comment]
Hey Charity, your HTML and CSS is on point. Your PR descriptions could be a bit more descriptive though, like for example list the details for - **How should this be manually tested?**  and add the navigation command.

**Pull request merged**